lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -84,6 +84,22 @@ import { ClearCacheRequest } from "../request/ClearCacheRequest";
 import { createNewGuid } from "../crypto/BrowserCrypto";
 import { initializeSilentRequest } from "../request/RequestHelpers";
 
+function getAccountType(
+    account?: AccountInfo
+): "AAD" | "MSA" | "B2C" | undefined {
+    const idTokenClaims = account?.idTokenClaims;
+    if (idTokenClaims?.tfp || idTokenClaims?.acr) {
+        return "B2C";
```

Comment:
Do we ever expect this condition to be hit when collecting telemetry? AFAIK no 1P apps are using B2C

Reply:
We do not expect this condition to be hit for 1P apps. However, i would prefer to handle this scenario for 3P apps injecting custom performance clients.